plutus-scb/app/Main.hs

```diff
@@ -286,11 +291,28 @@ reportContractHistoryParser =
         (ReportContractHistory <$> contractIdParser)
         (fullDesc <> progDesc "Show the state history of a smart contract.")
 
+data AppMsg =
+    InstalledContractsMsg
+    | ActiveContractsMsg
+    | TransactionHistoryMsg
+    | ContractHistoryMsg
+    | ProcessInboxMsg
+    | ProcessAllOutboxesMsg
```

Comment:
I kind of like the fact that looking at this makes it obviously suspicious that none of them have any data attached!

Reply:
yeah I think some of these should actually be used with `Observe` instead of `Log`. I'll make an issue for that.

Reply:
https://jira.iohk.io/browse/SCP-934